src/app/news/news-list-item.component.html

```diff
@@ -7,6 +7,7 @@
           item.doc.user.firstName + ((' ' + item.doc.user.middleName) || '') + ' ' + item.doc.user.lastName :
           item.doc.user.name}}
       </a>
+      <span *ngIf=!isActiveMember>(no longer member)</span>
```

Comment:
You should change this to `*ngIf="notMember"`.  It reads a little better than having `!` and having to mentally flip the boolean.